.travis.yml

```diff
@@ -136,15 +134,15 @@ matrix:
         - COVERAGE=Y
         - NO_HSTORE=Y
         - CREATE_PLPGSQL=Y
-    - jdk: oraclejdk8 # this will be overwritten by before_install above
+    - jdk: oraclejdk9
+      sudo: required
+      dist: trusty
+      group: edge
       addons:
         postgresql: "9.4"
-        apt:
-          packages:
-            - oracle-java9-installer
       env:
         - PG_VERSION=9.4
-        - JDK=9
+        - MCENTRAL=Y
```

Comment:
Is this an intentional change? As far as I remember, MCENTRAL=Y means "publish the snapshot to Maven Central". Our snapshots are produced by JDK6, JDK7, and JDK8. We don't yet have a special snapshot version for JDK9